src/Crypto/RNCryptor/V3/Decrypt.hs

```diff
@@ -94,6 +92,13 @@ removePaddingSymbols input =
   in B.take (B.length input - fromEnum lastWord) input
 
 --------------------------------------------------------------------------------
+decrypt_ :: AES128 -> ByteString -> ByteString -> ByteString
```

Comment:
Are you sure it's `AES128`? Again, I cannot see any mention to `AES128` anywhere in the format documentation, but maybe I'm missing something!